scripts/start.js

```diff
@@ -33,7 +34,8 @@ compiler.plugin('done', function (stats) {
 
     console.log(chalk.green('Compiled successfully!'));
     console.log('\nThe app is running at:');
-    console.log('\n    ' + chalk.cyan('http://localhost:' + port + '/'));
+    // TODO: display the correct port, if the default is busy.
+    console.log('\n    ' + chalk.cyan('http://localhost:' + DEFAULT_PORT + '/'));
```

Comment:
got it!
Ok, I think all of that looks very tricky.  What you think about such realization? https://gist.github.com/fobos/41058aa7c6fbf849908713e82520a7e7